tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Visit https://aka.ms/tsconfig to read more about this file */
     /* Projects */
     // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
-    // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
+    "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
```

Comment:
nit: fix offset of inline comment, which is not matching previous and next comments after your change

same below

Reply:
Maybe at some point will fix the formatting of this file, or even better, will remove all commented lines, but don't want to do it now

Reply:
Fair enough, but you're changing this line, which makes you the owner of the **line**, if not the file. You're leaving it in a worse state than it was, by not maintaining the consistency : - )